styles/styles.css

```diff
@@ -541,6 +541,10 @@ main .section.bold-headings h6 {
 }
 
 /* section - centered */
+.section.centered {
+  padding-inline: 1rem;
```

Comment:
There's a `padded-inline` section variant which you may want to check. I think the `centered` variant should be limited to styling needed for centring of content.